es/lecciones/transformacion-datos-xml-xsl.md

```diff
@@ -514,7 +514,6 @@ Para eliminar la sangría del texto, necesitarás hacerte cargo directo del espa
 ## Bibliografía recomendada por el traductor
 
 - Hunter, David *et al*. *Beginning XML*. Indianapolis: Wiley Publishing, 2007 (cuarta edición). Impreso.
-- Riley, Jenn. [*Understanding Metadata: What is Metadata, and What is For?*](https://www.niso.org/publications/understanding-metadata-2017) NISO, 2017. Web.
```

Comment:
Hello @acrymble. You could consider replacing the link with this one, which directs to the article as a PDF: https://groups.niso.org/apps/group_public/download.php/17446/Understanding%20Metadata.pdf

or this one, https://www.niso.org/node/12366, which directs to the abstract.

What do you think?

Reply:
Good suggestion. I've made the change.